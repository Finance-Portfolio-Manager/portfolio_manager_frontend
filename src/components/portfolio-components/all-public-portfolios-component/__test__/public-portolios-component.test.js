import React, { Component } from 'react';
import ReactDOM from 'react-dom';
import PublicPortfolios from './../public-portfolios-component';

import { render, cleanup } from '@testing-library/react';
import "@testing-library/jest-dom/extend-expect";

import renderer from "react-test-renderer";

import { Switch, Route, BrowserRouter as Router } from 'react-router-dom';

import { shallow, ShallowWrapper, mount } from "enzyme";

import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });
const wrapper = shallow(<PublicPortfolios></PublicPortfolios>);

test("matches snapshot", ()=>{
    const tree = renderer.create(<PublicPortfolios></PublicPortfolios>).toJSON();
    expect(tree).toMatchSnapshot();
})