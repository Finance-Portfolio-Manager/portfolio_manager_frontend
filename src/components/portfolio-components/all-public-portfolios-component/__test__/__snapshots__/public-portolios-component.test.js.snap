// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#B9B9BA",
    }
  }
>
  <div
    className="container"
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "justifyContent": "center",
      }
    }
  >
    <div
      className="row"
    >
      <h2>
        Public Portfolios
      </h2>
      <div
        className="row py-3"
        style={
          Object {
            "borderStyle": "solid",
            "width": "70rem",
          }
        }
      />
    </div>
  </div>
</div>
`;
